Sources/Crypto/RSA/RSAKey.swift

```diff
@@ -102,6 +107,35 @@ final class CRSAKey {
         self.pointer = pointer
     }
 
+    /// Creates a new DER encoded `CRSAKey` from data.
+    static func make(from der: Data) throws -> CRSAKey {
+        guard let derDecoded = Data(base64URLEncoded: der) else {
+            throw CryptoError.openssl(identifier: "rsax509", reason: "Certificate decoding failed")
+        }
+        var bytes = [UInt8]()
+        bytes.append(contentsOf: derDecoded)
+        guard let x509 = bytes.withUnsafeBufferPointer({ rawKeyPointer -> UnsafeMutablePointer<X509>? in
```

Comment:
You shouldn't need the intermediary `[UInt8]` here, `Data` has methods for accessing a buffer pointer.